perfdash/Makefile

```diff
@@ -1,7 +1,7 @@
 all: push
 
 # See deployment.yaml for the version currently running-- bump this ahead before rebuilding!
-TAG = 2.37
+TAG = 2.38
 
 REPO = gcr.io/k8s-testimages
```

Comment:
Could you also change repo to k8s-staging-perf-tests, please?

Reply:
Done